app/config/config.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const localEnv = _loadLocalEnv(),
+    env = process.env.NODE_ENV || 'development';
+
+
+/**
+ * @property {string} HOST
+ * @property {string} PORT
+ * @property {string} MONGODB_URL
+ */
+const config = {
+
+    development: {
+        HOST: localEnv.HOST || '188.166.63.248',
+        PORT: localEnv.PORT || '9030',
+
+        MONGODB_URL: localEnv.MONGODB_URL || 'mongodb://localhost:27017/differ-canvas'
+    },
+
+    test: {
+        HOST: '127.0.0.1',
+        PORT: '3000',
+
+        MONGODB_URL: 'mongodb://localhost:27017/differ-canvas_test_integration'
+    },
+
+    production: {
+        HOST: '127.0.0.1',
+        PORT: '9010',
+
+        MONGODB_URL: 'mongodb://localhost:27017/differ-canvas'
+    }
+};
+
+
+module.exports = config[env];
+
+
+/**
+ * @type function
```

Comment:
Let me be against. With the same logic we should bring here the logic of getCourses service function, because the main logic of the route goes here.
I tried to break the app into logical components and middleware is something to be separated. Image, I have 10 middlewares for with route (checking input, does the user loged in, does the requested resource belongs to the user, ...). 10 middlewares putting in the route is gonna make a mess. On the other hand, I can list like:
CanvasMiddle.validateGetCoursesArgs,
Authenticate,
IsResourceBelongsTo,
....